server/src/Models/DB.hs

```diff
@@ -121,7 +129,7 @@ Review
 
 
 Cart
-    customerId CustomerId Maybe
+    customerId CustomerId Maybe OnDeleteCascade
```

Comment:
Could you elaborate on why this change was added?

Reply:
I did this change for test purpose only: I wanted to delete existing account to register one more time to test the flow. Reverted now.

Reply:
FWIW, the admin panel has functionality for "deleting" the customer (by merging it into a dedicated "deleted" customer)